src/vs/workbench/parts/extensions/browser/media/extensionActions.css

```diff
@@ -65,4 +65,8 @@
 .hc-black .extensions-viewlet>.extensions .extension>.details>.footer>.monaco-action-bar .action-item .action-label.extension-action.manage,
 .vs-dark .extensions-viewlet>.extensions .extension>.details>.footer>.monaco-action-bar .action-item .action-label.extension-action.manage {
 	background: url('manage-inverse.svg') center center no-repeat;
-}
\ No newline at end of file
+}
+
+.extensions-viewlet.dragged-over {
+	background-color: rgba(255, 255, 255, 0.5);
```

Comment:
Have to careful... not sure if it works in all themes which have different back ground colours.
Try adding only opacity.